WhatsNew.Cli/README.md

```diff
@@ -29,51 +29,51 @@ Metrics for the "what's new" pages are made available at [aka.ms/whatsnewindocs]
     - For public repos:
       - **repo** > **public_repo**
       - **admin:org** > **read:org**
-1. For private repos in the *MicrosoftDocs* GitHub organization, after generating the PAT, select **Enable SSO** and choose **Authorize** for both **microsoft** and **MicrosoftDocs**.
+1. For private repos in the *MicrosoftDocs* GitHub organization, after generating the PAT, select **Enable SSO** and choose **Authorize** for the **MicrosoftDocs**.
 1. Store the PAT in an environment variable named `GitHubKey`.
 1. Set up an OSPO personal access token
    > This step is required for versions 2.0 and above.
 
    The OSPO personal access token is required to access the OSPO API. This API determines community contributors and Microsoft employee and vendor contributors.
 1. Request a token at [Visual Studio Online](https://ossmsft.visualstudio.com/_usersSettings/tokens). You can disable all scopes *except* read:user profile.
-1. Store the token in an environment variable named "OSPO_KEY".  If you are using the GitHub Action to generate the PR automatically, add the key as a secret in your repository:
+1. Store the token in an environment variable named "OSPO_KEY". **If you are using the GitHub Action to generate the PR automatically**, add the key as a secret in your repository:
    - Go to **Settings** on your repo.
    - Select **Secrets**
    - Add "OSPO_KEY" as a **Repository Secret**.
 1. Install the [.NET SDK 7.0](https://dotnet.microsoft.com/download/dotnet) or later.
-1. build and publish the tool:
+1. build and publish the tool. The `WhatsNew.Cli` folder is at the root of the repository you closed in step 1.
 
     ```bash
```

Comment:
Does this build debug or release by default?  I assume it builds debug since that's what the instructions have your run, after.

Reply:
Yes, it's debug. Added a note.